CatAnalyzer/test/run_TtbarDiLeptonAnalyzer_cfg.py

```diff
@@ -9,7 +9,8 @@
 
 process.source = cms.Source("PoolSource", fileNames = cms.untracked.vstring())
 from CATTools.Validation.commonTestInput_cff import commonTestCATTuples
-process.source.fileNames = commonTestCATTuples["sig"]
+process.source.fileNames = ["root://cms-xrdr.sdfarm.kr:1094///xrd/store/group/CAT/TT_TuneCUETP8M2T4_13TeV-powheg-pythia8/v8-0-4_RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/170113_045423/0000/catTuple_983.root"]
+#process.source.fileNames = commonTestCATTuples["sig"]
```

Comment:
My idea here was to use the common root file for the testing. Is there any special reason to use this root file? To pick up v804?